Dockerfile

```diff
@@ -145,8 +144,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         php7.4-zip \
         php7.4-intl \
         pngcrush \
-        python-setuptools \
```

Comment:
I responded to an outdated comment, but I think we should keep setuptools, might be useful for some customers and it's pretty popular.  any reason why we're removing it?

Reply:
I didn't have a particular reason for removing it so I'll leave it in.

Reply:
kewl, i think we should, as long as we still support python 2.7